paasta_tools/kubernetes_tools.py

```diff
@@ -246,6 +246,10 @@ def copy(self) -> "KubernetesDeploymentConfig":
     def get_bounce_method(self) -> str:
         """Get the bounce method specified in the service's kubernetes configuration."""
         # map existing bounce methods to k8s equivalents.
+        # but if there's an EBS volume we must downthenup to free up the volume.
+        # in the future we may support stateful sets to dynamically create the volumes
+        if self.get_aws_ebs_volumes():
+            return 'Recreate'
```

Comment:
KUBE_DEPLOY_STATEGY_MAP['downthenup']

just to be consistent with the line below

or even:

    if self.get_aws_ebs_volumes():
        strategy = 'downthenup'
    else:
        strategy = self.config_dict.get('bounce_method', 'crossover')
    return KUBE_DEPLOY_STATEGY_MAP[strategy]